weather_dl_v2/cli/app/services/license_service.py

```diff
@@ -48,6 +48,13 @@ def _remove_license(self, license_id: str):
     def _update_license(self, license_id: str, license_dict: dict):
         pass
 
+    @abc.abstractmethod
+    def _redeploy_license_by_license_id(self, license_id: str):
+        pass
+
+    @abc.abstractmethod
+    def _redeploy_licenses_by_client(self, license_id: str):
```

Comment:
`client_name`.

Reply:
fixed in [e892514](https://github.com/google/weather-tools/pull/420/commits/e892514e70b7ed5bc1509848f9d36eca289ec872)